src/Kiota.Builder/Extensions/OpenApiUrlTreeNodeExtensions.cs

```diff
@@ -131,9 +134,11 @@ private static string GetSegmentName(this OpenApiUrlTreeNode currentNode, HashSe
     public static string CleanupDescription(this string? description) => string.IsNullOrEmpty(description) ? string.Empty : descriptionCleanupRegex.Replace(description, string.Empty);
     public static string GetPathItemDescription(this OpenApiUrlTreeNode currentNode, string label, string? defaultValue = default) =>
     !string.IsNullOrEmpty(label) && (currentNode?.PathItems.ContainsKey(label) ?? false) ?
-            (currentNode.PathItems[label].Description ??
-            currentNode.PathItems[label].Summary ??
-            defaultValue).CleanupDescription() :
+            (currentNode.PathItems[label].Description is string description && !string.IsNullOrEmpty(description) ?
```

Comment:
I think `currentNode.PathItems[label].Description is string` is always true here based on the declared types.

Reply:
no, it could be null. https://dotnetfiddle.net/yZmVBT